usr/share/rear/conf/default.conf

```diff
@@ -1996,9 +1996,39 @@ NSRSERVER=
 NSR_RETENTION_TIME=
 # The name of the default NSR pool. Default is the upstream default but some sites need to change that to an own default pool name.
 NSR_DEFAULT_POOL_NAME=Default
-# The NSR_CLIENT_MODE defines whether you have direct access to the NSRSERVER's recovery or are just a passive NSR client
-# waiting for the files to be recovered from the NSRSERVER (default is "no")
+# There are different EMC Legato server based recovery options which highly depend on
+# the client's permissions granted on the EMC Networker (Legato) server side.
+# 
+# Either it is allowed to self-service a recovery job on a EMC Networker server
+# (i.e. direct access) or it is required to trigger someone else who is capable
+# to start the file recovery job for the client (i.e. client does not have direct
+# access to the EMC Networker server).
+#
+# Those recovery actions are covered with the following settings:
+# NSR_CLIENT_MODE = YES           ; Client needs to trigger someone else to start the 
```

Comment:
Only a note:
Perhaps it could be better to use bash variable assignment syntax
also in comments e.g. something like
```
# NSR_CLIENT_MODE=yes           ; Client needs to trigger someone else to start the 
#                                 EMC Networker server recovery action
# or
# NSR_CLIENT_MODE=no (or unset) ; Default - We have full control
...
# NSR_CLIENT_REQUESTRESTORE=yes          ; Client needs to request for a recovery action 
#                                          (is NOT allowed to perform recovery actions on its own).
# or
# NSR_CLIENT_REQUESTRESTORE=no (or unset); Default - Client does NOT need to request for a
#                                          recovery action (is allowed to perform recovery
#                                          actions on its own).
```
(i.e. no spaces around the '=') so that copy&paste works?
On the other hand having spaces around the '=' in comments aids readability.

Reply:
It is up to You to decide what is better for the user. This change would be a minor. :-)

Reply:
It is your code so the decision is up to you.

Reply:
I applied the modifications proposed.